plugins/com.google.cloud.tools.eclipse.util/templates/appengine/MockHttpServletResponse.java.ftl

```diff
@@ -0,0 +1,174 @@
+<#if package != "">package ${package};
+
+</#if>import java.io.IOException;
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.util.Locale;
+
+import javax.servlet.ServletOutputStream;
+import javax.servlet.http.Cookie;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * This class is created only for the purpose of demonstrating unit testing of the
+ * {@link HelloAppEngine} class and the only those methods have meaningful implementation
+ * which are use in the unit test.
```

Comment:
Splitting the sentence makes sense, also a `d` is missing from `are use in` (should be `are used in`). But otherwise I don't see the improvement over my version (which can be due to me being non-native speaker).